javascript-functions/main.js

```diff
@@ -0,0 +1,32 @@
+function convertMinutesToSeconds(minutes) {
+  const seconds = minutes * 60;
+  return seconds;
+}
+console.log('5 minutes converted to seconds is:', convertMinutesToSeconds(5));
+
+function greet(name) {
+  return '"Hey, ' + name + '"';
+}
+console.log(greet('Beavis'));
+
+function getArea(width, height) {
+  return width * height;
+}
+console.log('The total area is:', getArea(17, 42));
+
+function getFirstName(person) {
+  return person.firstName;
+}
+console.log(
+  'The first name is:',
+  getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' })
+);
+
+function getLastElement(array) {
+  const lastElement = array[3];
```

Comment:
Instead of hard coding `3` as the index. You want to utilize the `length` property of `array` in order to access the last value. What happens if we add another value to the array? Then it wouldn't be returning the last one